commit 2e8f667e6c2f2c6343c38b6b294ca1db99cb0c80
Author: Lewin Bormann <lewin.bormann@zhinst.com>
Date:   Wed Jun 11 16:21:12 2025 +0200

    Use kj::max instead of std::max in kj/timer.c++

diff --git a/c++/src/kj/timer.c++ b/c++/src/kj/timer.c++
index 03526c54..98c47491 100644
--- a/c++/src/kj/timer.c++
+++ b/c++/src/kj/timer.c++
@@ -112,10 +112,10 @@ Maybe<uint64_t> TimerImpl::timeoutToNextEvent(TimePoint start, Duration unit, ui
 void TimerImpl::advanceTo(TimePoint newTime) {
   // It has been observed that clock_gettime 
   // may return non monotonic time, even when CLOCK_MONOTONIC is used.
-  // We use std::max to guard against this rare issue.
+  // We use kj::max to guard against this rare issue.
   // - on mac: https://github.com/capnproto/capnproto/issues/1693
   // - on linux: https://github.com/capnproto/capnproto/issues/2261
-  time = std::max(time, newTime);
+  time = kj::max(time, newTime);
 
   for (;;) {
     auto front = impl->timers.begin();
